tests/unit/test_prometheus_schema_contract.py

```
@@ -0,0 +1,320 @@
+        self.assertGreaterEqual(checked_ratio, 0.0)
+        self.assertLessEqual(checked_ratio, 1.0)
+
+
+class TestPrometheusDocsAndAlertsContract(unittest.TestCase):
+    def test_readme_contains_v2_contract_and_dns_note(self):
+        readme = "/workspace/docs/prometheus/README.md"
+        with open(readme, encoding="utf-8") as f:
+            text = f.read()
+
+        # Schema + state model
+        self.assertIn("**Schema:** v2", text)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)